DataDog/chaos-controller
# remove unnecessary replaces from go.mod

## What does this PR do?

- [ ] Adds new functionality
- [ ] Alters existing functionality

A brief description of changes to implementation or controller behavior:

I introduced a circular dependency some months back thinking it was necessary but after studying up on packages, I think it should be removed.

## Code Quality

### Testing

- [ ] I used existing unit tests
- [x] I manually tested locally
- [ ] I will manually test in a canary deployment

1. I deleted a bunch of modules and ran:

* make minikube-build
* make install
* make restart

I was unable to build, confirming that this set of commands does pick up `go.mod` changes.

2. I put back all of the other modules and only deleted the `replace` lines.

* make minikube-build
* make install
* make restart

This time, I was able to create a example demo set, create a disruption, verify that it's working, and then clean everything up.

### Checklist

- [x] My code is sufficiently commented
- [x] I have tested to the best of my ability
- [x] I have signed my commit (see [Contributing Docs](../CONTRIBUTING.md))
